alignment.py

```diff
@@ -0,0 +1,67 @@
+import cv2 as cv
+import numpy as np
+import math
+
+def readKeyPointsFromFile(path):
+    with open(path, 'r') as f:
+        lines = [line.rstrip() for line in f]
+    f.close()
+    keyPoints = list()
+    for line in lines:
+        point = list(map(int, line.split()))
+        keyPoints.append(cv.KeyPoint(point[0], point[1], 1))
+    return keyPoints
+
+def countPixelCoordinates(imageSource, pixel, destinationImage, homographyMatrix, coordinatesOfTopLeft,
```

Comment:
it's to better to call it `calculate`

Reply:
![reprojection_error_esri](https://github.com/AnNyiiik/AlignmentOfUAVImages/assets/114094098/5c72f6f3-a1da-4fde-b590-43fd5adeb44c)
![google_satellite_error_pandas](https://github.com/AnNyiiik/AlignmentOfUAVImages/assets/114094098/cf6637a2-196b-4c02-b08f-d832a3cb4088)